### PR TITLE
Vcf qio

### PR DESCRIPTION
**Pull request by ChristinaXu2017**
# Description
- Replace old VCFFilerReader with new VcfFileReader in adamajava projects.
- Replace old VCFFileWriter with org.qcmg.qio.record.RecordWriter in adamajava projects
- updated projects: q3panel, qpileup, qmule, q3vcftools, qprofiler, qprofiler2, qsignature, qsnp. 
- delete org.qcmg.vcf package;
- tidy up qcommon, such as commented code, space. 

## Type of change
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
In this pull request, only VCFFileReader and VCFFileWriter are replaced, This Reader and Writer are well tested in the previous pull request with a real WGS data set (#243, #245).  Here we also compare the output from qprofiler, qprofiler2, qsignature, qsnp to adamajava production, the outputs are identical (/mnt/lustre/working/genomeinfo/share/testdataXu/<project>/20210222).  

related unit tests are also updated.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by holmeso** [comment]
As a general note, in the _How Has This Been Tested?_ section, we are looking for tests that have been performed during development/testing. The TC regression tests are run after a change has been merged into master and so have not been run yet on the proposed code changes.

File: q3panel/src/au/edu/qimr/panel/Q3ClinVar.java [hunk 1]
```diff
@@ -483,22 +481,22 @@ private void loadTiledAlignerData() throws Exception {
 		
 		logger.info("loading genome tiles alignment data");
 		
-		try (TabbedFileReader reader = new TabbedFileReader(new File(refTiledAlignmentFile))) {
+		try (StringFileReader reader = new StringFileReader(new File(refTiledAlignmentFile))) {
 			
-			TabbedHeader header = reader.getHeader();
+			List<String> header = reader.getHeader();
```
[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply q3panel/src/au/edu/qimr/panel/Q3ClinVar.java:486]: They are the same, but TabbedFileReader is replacing by StringFileReader; TabbedHeader is also replacing by List<String>. TabbedHeader, TabbedFileReader etc will be deleted on the next pull requests.

[the remark this answers is not in the document]
Comment (holmeso) [reply q3panel/src/au/edu/qimr/panel/Q3ClinVar.java:486]: If `List<String> header` and `List<String> headerList` are the same, then I would delete one of them

[the remark this answers is not in the document]
Comment (ChristinaXu2017) [reply q3panel/src/au/edu/qimr/panel/Q3ClinVar.java:486]: yes, you are right. they are deleted now.

**Comment by ChristinaXu2017** [comment]
tested on WGS dataset for qprofiler, qprofiler2, qsignature, qsnp.

**Review by holmeso — APPROVED** [review]
looks good

**Pull request merged**